helm/rbac-operator/values.yaml

```diff
@@ -9,7 +9,9 @@ pod:
     id: 1000
 
 registry:
-  domain: quay.io
+  domain: docker.io
```

Comment:
Also changing to docker.io here as I believe they should all be docker by default at this point